### PR TITLE
[OFFAPPS-863] V2 migration

### PR DESCRIPTION
**Pull request by ocke**
/cc @vaevictis 
/cc @sostopher 

First app converted!

Review by commit for easy reviewing

Jira: https://zendesk.atlassian.net/browse/OFFAPPS-863

### CODE REVIEW

**Comment by sostopher** [comment]
@ocke Can this be merged and go to QA now? Anything else to be done?

File: assets/main.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+client.on('ticket.assignee.user.email.changed', app.renderText);
+client.on('ticket.requester.id.changed', app.renderText);
+client.on('ticket.requester.name.changed', app.renderText);
+client.on('ticket.requester.email.changed', app.renderText);
+client.on('ticket.type.changed', app.renderText);
+client.on('ticket.subject.changed', app.renderText);
+client.on('ticket.priority.changed', app.renderText);
+client.on('ticket.status.changed', app.renderText);
+client.on('ticket.tags.changed', app.renderText);
+client.on('ticket.collaborators.changed', app.renderText);
```
Comment (danielbreves) [inline assets/main.js]:
```
[
/// ...
].forEach(app.renderText)
```

File: settings.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+textBody: |
```
Comment (danielbreves) [inline settings.yml]: How does this work?

Comment (ocke) [reply settings.yml]: that means it's multiline in yaml

Comment (danielbreves) [reply settings.yml]: I meant the YML file, but I remembered you introduced this here: https://developer.zendesk.com/apps/docs/developer-guide/setup#defining-test-settings.

File: assets/main.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+      }
+      return string || '';
+
+    } catch(e) {
+      console.error(placeholderStripped + ' cannot be interpolated');
+    }
+  },
+
+  getData: function(placeholderStripped) {
+    return client.get(['ticket', 'currentUser', 'ticket.assignee']).then(function(data) {
```
Comment (danielbreves) [inline assets/main.js]: Isn't `ticket.assignee` already part of `ticket`?

Comment (ocke) [reply assets/main.js]: strange enough, that doesn't work. assignee in the activity is async?

Comment (danielbreves) [reply assets/main.js]: Yep, that's a bug with the legacy API router.

**Pull request merged**